### ISSUE TITLE
Should warn on property accessor call on in parameter

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#49456]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/content/idea/726523/should-warn-on-property-accessor-call-on-in-parame.html)._

---
Consider this:

```csharp
struct MyStruct
        {
            public int Property { get; set; }

public void Test() {  }
        }

void Method(in MyStruct myStruct)
        {
            // warning
            myStruct.Test();

// no warning???
            var x = myStruct.Property;
        }
```

---
#### Jane Wu [MSFT] on 9/9/2019, 00:00 AM: 

<p>Thank you for taking the time to provide your suggestion. We will do some preliminary checks to make sure we can proceed further.  We’ll provide an update once the issue has been triaged by the product team.</p>

**Comment by sharwell** [comment]
😕 I don't see a warning for `myStruct.Test()`. I'm guessing this is a feature request for an analyzer that detects copies of `ref readonly` values, but without any indication of which analyzer is in use I'm not sure we can route this to the correct team.
